.github/workflows/deploy_to_staging.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will update the code for the staging environment
+
+on:
+  pull_request:
+  push:
+    branches: ["main"]
+
+name: Deploy to staging
+
+jobs:
+  build_and_deploy:
+    name: Deploy
+    environment:
+      name: Staging
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+
+      - name: Set up SAM
+        uses: aws-actions/setup-sam@v1
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ap-south-1
+
+      - name: Build
+        run: sam build --use-container -t templates/staging.yaml
+
+      - name: Deploy
+        env:
+          JWT_SECRET_KEY: ${{ secrets.JWT_SECRET_KEY }}
```

Comment:
Same JWT key for staging and production? Maybe one of them should be different?

Reply:
Done.